x/gamm/types/pool.go

```diff
@@ -262,6 +263,12 @@ func (pa *Pool) setInitialPoolParams(params PoolParams, sortedAssets []PoolAsset
 			params.SmoothWeightChangeParams.StartTime = time.Unix(curBlockTime.Unix(), 0)
 		}
 	}
+
+	// Set pool start time if not present.
+	if params.StartTime.Unix() <= 0 {
```

Comment:
Is there any case Unix is less than 0?

Reply:
Yes, thats how you refer to times before the Unix start time (Jan 1 1970)

Reply:
@mconcat what happens here if the start time wasn't set? Perhaps we should have a test for unset start time?